.werft/observability/monitoring-satellite.ts

```diff
@@ -191,3 +183,29 @@ function ingressReady(namespace: string, name: string): boolean {
     werft.log(sliceName, `${name} ingress not ready.`)
     return false
 }
+
+/**
+ * installSetup creates or replaces everything that is needed to run monitoring-satellite.
+ * This setup includes Custom Resource Definitions(CRDs), kubernetes operators and namespaces.
+ */
+function installSetup() {
+    // Adds a label to the namespace metadata.
+    // This label is used by ServiceMonitor's namespaceSelector, so Prometheus
+    // only scrape metrics from its own namespace.
+    exec('kubectl apply -f observability/monitoring-satellite/manifests/namespace.yaml', {silent: true})
+    exec('kubectl apply -f observability/monitoring-satellite/manifests/podsecuritypolicy-restricted.yaml', {silent: true})
+
+    let crdExists = exec('kubectl get customresourcedefinitions.apiextensions.k8s.io alertmanagers.monitoring.coreos.com', {silent: true}).code == 0
```

Comment:
It is preferred to use `const` whenever a variable never needs to be assigned :)